Saasu.API.Client.IntegrationTests/packages.config

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.AspNet.WebApi.Client" version="5.2.3" targetFramework="net471" />
-  <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net471" />
+  <package id="Newtonsoft.Json" version="10.0.3" targetFramework="net461" />
```

Comment:
Why are you downgrading the target framework?

Reply:
As above